src/components/@core/navbar/index.tsx

```diff
@@ -18,10 +18,10 @@ export default function NavBar() {
         backdropFilter="saturate(180%) blur(5px)"
         bg="whiteAlpha.800"
         id="menu"
-        position="fixed"
         width="100%"
         zIndex={4}
         shadow="md"
+        position={"fixed"}
```

Comment:
How is `position="fixed"` and `position={"fixed"}` are different?

Reply:
Both are same , we can use both.